### DISCUSSION TITLE
iOS App

### DISCUSSION THREAD

**Discussion by aniruddha-adhikary** [aniruddha-adhikary/mrt-buddy#31] [Polls]
So a lot of folks are asking for an iOS App. What should be our approach?

I feel having a separate codebase for iOS opens up maintainability and feature-parity issues. Also rewriting the app in flutter may make the contributors... well... unfamiliar with the codebase.

**Comment by hmasum52** [comment]
I think maintaining two separate code base would be very difficult!

For framework I think Flutter may be a better choice than React Native for some projects because it offers more flexibility and control over the user interface (UI), and it's faster and performs better as it doesn't use a JavaScript bridge.
